extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -117,28 +117,32 @@ function render_player( $player_data, $attributes ) {
 
 	ob_start();
 	?>
-	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
-		<noscript>
-			<ol class="jetpack-podcast-player__episodes">
-				<?php foreach ( $player_data['tracks'] as $attachment ) : ?>
-				<li class="jetpack-podcast-player__episode">
-					<a
-						class="jetpack-podcast-player__episode-link"
-						href="<?php echo esc_url( $attachment['link'] ); ?>"
-						role="button"
-						aria-pressed="false"
-					>
-						<span class="jetpack-podcast-player__episode-status-icon"></span>
-						<span class="jetpack-podcast-player__episode-title"><?php echo esc_html( $attachment['title'] ); ?></span>
-						<time class="jetpack-podcast-player__episode-duration"><?php echo ( ! empty( $attachment['duration'] ) ? esc_html( $attachment['duration'] ) : '' ); ?></time>
-					</a>
-				</li>
-				<?php endforeach; ?>
-			</ol>
-		</noscript>
+	<div class="<?php echo esc_attr( $block_classname ); ?> is-default" id="<?php echo esc_attr( $instance_id ); ?>">
```

Comment:
I think you could add `is-default` to `block_classes` directly and let `block_classes` handle all classes for you.

`$block_classname = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attributes, array( 'is-default' ) );`

Reply:
on it